flow/rocev2_flow.yaml

```diff
@@ -0,0 +1,105 @@
+components:
+  schemas:
+    Rocev2.Flow:
+      description: |-
+        Rocev2 traffic flow configuration.
+      type: object
+      properties:
+        flow_infos:
+          $ref: '#/components/schemas/Rocev2.FlowInfo'
+          x-field-uid: 1
+        per_port_settings:
+          $ref: '#/components/schemas/Rocev2.PerPortSettings'
+          x-field-uid: 2
+    Rocev2.PerPortSettings:
+      description: |-
+        Rocev2 traffic flow configuration.
+      type: object
+      properties:
+        target_line_rate:
+          description: |-
+            Target Line Rate in percentage.
+          type: integer
+          format: uint64
+          x-field-uid: 1
+        inter_batch:
+          $ref: '#/components/schemas/Rocev2.InterBatch'
+          x-field-uid: 2
+    Rocev2.FlowInfo:
+      description: |-
+        Rocev2 Flow Info, per flow details.
+      type: object
+      properties:
+        name:
+          description: |-
+            Name of the flow group.
+          type: string
+          x-field-uid: 1
+        tx_endpoint: 
+          description: |-
+            Tx Endpoint. Empty for Rocev2.
```

Comment:
It is not clear what is meant by empty. Is it like RSVP-TE where the src defines the destination and must choose correct egress for specific ingress and no choice is available ? 
It is not clear to me how this will help user control which QPs to actually send traffic on if this is empty. If a field is of no value we need to removed it.